templates/worker/01-worker-container-runtime/_base/files/crio.yaml

```diff
@@ -1,97 +1,20 @@
 filesystem: "root"
 mode: 0644
-path: "/etc/crio/crio.conf"
+path: "/etc/crio/crio.conf.d/00-default"
```

Comment:
Is there even any difference between the worker/master files, or could this become a single file in `templates/common` now?

Reply:
This could, but I will consolidate it in a future PR